docker/entrypoint.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+if [[ $1 ]] # if there is an argument assume its for the cli
+then
+  # To get rid of version number, assumes there is only one jar in the directory
+  JAR_NAME=$(ls /usr/share/java/backup-cli/cli*.jar | head -1)
```

Comment:
This feels a bit hacky. I would rather rename the jar when it gets built or even better would be to create a native image using GraalVM. In this case I'd prefer the later. PicoCLI supports it as well:

https://picocli.info/#_graalvm_native_image

and the Gradle Plugin is here https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html